src/controllers/_file_controllers.py

```diff
@@ -18,3 +18,18 @@ def check_state_handler(token, file_uuid):
     except Exception as e:
         print("[Exception] check_state_handler ->", str(e))
         return {"msg": "There was an error checking the file state"}, 500
+
+
+def rename_handler(token, file_uuid, new_name):
+    try:
+        request_data = {"token": token, "fileUUID": file_uuid, "newName": new_name}
+        response = soap_client.service.file_rename(request_data)
+
+        has_success_code = str(response.code).startswith("20")
```

Comment:
I think you don't need to use the `startsWith("20")` validation. It was helpful in the endpoint to get a file status because it has two possible success codes (`200` and `204`), but this endpoint just has one possible success code (`204`). Just check the `response.success` field.